### PR TITLE
ms.service update.

### PR DESCRIPTION
**Pull request by brittmsantos**
## Summary

Fixed:

- ms.service values removed and/or updated to approved taxonomy values

This PR makes 1 change in 1 file. No other content has been edited or added.

This PR is part of a global effort to improve metadata quality across the Learn platform. These changes make reports cleaner, easier to use, more consistent, and more reflective of our current business. After you accept these changes and they’re merged, they will be reflected in standard dashboards and reports maintained by C+E Skilling Business & Customer Success. You’ll need to use new filter values in these reports, and you may need to update any custom dashboards or Kusto queries. If you have any questions, please contact Learn Platform Metadata ([learn-metadata@microsoft.com](mailto:learn-metadata@microsoft.com)).

### CODE REVIEW

File: docs/azure/landing-page.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
   ms.topic: landing-page
   ms.collection: collection
   ms.technology: azure
-  ms.service: multiple
+  ms.service: app-service
```
Comment (IEvangelist) [inline docs/azure/landing-page.yml:12]:
```suggestion
  ms.service: azure
```

I believe that `ms.service: azure` is more appropriate. This content is not limited to Azure App Service, but covers much more Azure offerings. See https://review.docs.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main#msservice

**Review by IEvangelist — CHANGES_REQUESTED** [review]

File: docs/azure/landing-page.yml (hunk 1 above)
Comment (brittmsantos) [reply docs/azure/landing-page.yml:12]: Hi @IEvangelist, `ms.service: azure` is scheduled for removal from the allowlist. You can contact the Learn Platform Metadata ([learn-metadata@microsoft.com](mailto:learn-metadata@microsoft.com)) for more information regarding the allowlist. In the meantime, I'd be happy to change this value to another approved one that is better suited for this content!

Comment (IEvangelist) [reply docs/azure/landing-page.yml:12]: Hi @brittmsantos - what about `cloud-adoption-framework`. That seems to cover a lot of the concepts driving this area of the content. It's all about how .NET can intise the adoption of Azure, no?

Comment (brittmsantos) [reply docs/azure/landing-page.yml]: That works for me! I've gone ahead and implemented this change.

**Comment by brittmsantos** [comment]
**Global effort to improve metadata quality across the Learn platform**

@darylmsft @adriankjohnson

These changes make reports cleaner, easier to use, more consistent, and more reflective of our current business. After you accept these changes and they're merged, they will be reflected in standard dashboards and reports maintained by C+E Skilling Business & Customer Success. You'll need to use new filter values in these reports, and you may need to update any custom dashboards or Kusto queries. If you have any questions, please contact Learn Platform Metadata ([learn-metadata@microsoft.com](mailto:learn-metadata@microsoft.com)).

**Review by IEvangelist — APPROVED** [review]

**Pull request merged**